packages/plugin-bibtex/test/output.json

```diff
@@ -68,7 +68,11 @@
     "label": [[
       {"id": "b", "citation-label": "foo", "type": "book"},
       {"id": "b", "author": [{"given": "foo"}], "type": "book"},
-      {"id": "b", "type": "book"}
+      {"id": "b", "type": "book"},
+      {"id": 123, "type": "book"},
+      {"id": 0, "type": "book"},
+      {"type": "book"},
+      {"id": null, "type": "book"}
```

Comment:
Any idea why the syntax highlighter is representing this as "invalid.illegal"? The tests are passing so it should be valid JSON...

![Screenshot_20211129_202112](https://user-images.githubusercontent.com/14018963/143929384-c158b73a-43e1-4783-b6f1-66aa649ddc55.jpg)